src/Content/Backend/Solution/Monaco.Template.Backend.Domain.Tests/ImageTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Monaco.Template.Backend.Domain.Model;
+using System.Diagnostics.CodeAnalysis;
+using System.Security.Cryptography;
+using FluentAssertions;
+using Monaco.Template.Backend.Common.Tests.Factories;
+using Xunit;
+
+namespace Monaco.Template.Backend.Domain.Tests;
+
+[ExcludeFromCodeCoverage]
+[Trait("Core Domain Entities", "Image Entity")]
+public class ImageTests
+{
+	[Theory(DisplayName = "New Image succeeds")]
+	[AnonymousData]
+	public void NewImageSucceeds(Guid id,
+									string name,
+									string extension,
+									long size,
+									string contentType,
+									bool isTemp,
+									int height,
+									int width,
+									DateTime dateTaken,
+									Image thumbnail)
+	{
+		var latitude = RandomNumberGenerator.GetInt32(-90, 90);
+		var longitude = RandomNumberGenerator.GetInt32(-180, 180);
+
+		var sut = new Image(id,
+							name,
+							extension,
+							size,
+							contentType,
+							isTemp,
+							height,
+							width,
+							dateTaken,
+							latitude,
+							longitude,
+							thumbnail);
+
+		sut.Id
```

Comment:
Personal opinion, so feel free to disagree but I find it harder to read these when they are spread across lots of new lines and each assertion is not separated by a new line. However my preference would be to just keep each assertion on a single line if it is short.

I'd add that I think the benefit of libraries like FluentAssertions is that they allow us to write code that reads like a sentence and when that sentence is short enough it can be easier to read it left to right than breaking it across multiple lines and having to scan top to bottom as well as left to right.

Reply:
Yeah, I think that's the R# formatting kicking in to format this way when I have already methods chained together. Will update to keep to other format you mention on these assertions.

Reply:
The only problem with this format (and going back in time I think it was another reason to keep it this way) is when I have many more methods like in some places when I also had an And.SomethingElse(moreParams)... Stuff like that...
At that point it becomes easier to read just in a "column" of stuff separated by lines because otherwise is having a lot of things mixed up in the same line, matching just the initial field and Should() call. I remember being more a consistence thing so everything would be on the same style.

What do you think?

Reply:
I completely agree, there is a width when a line is no longer 'short' and should be broken up across multiple lines, I think we get this right elsewhere in the project but I'm not sure it's right to break short statements onto new lines in all cases.

For example if we look at:

```csharp
result.Should().NotBeNull();
result!.Pager.Count.Should().Be(companiesSet.Count);
result.Items.Should().HaveCount(companiesSet.Count).And.Contain(x => companiesSet.Any(c => c.Name == x.Name)).And.BeInDescendingOrder(x => x.Name);
```

It could be formatted in a number of ways including:
A) 

```csharp
result.Should().NotBeNull();
result!.Pager.Count.Should().Be(companiesSet.Count);
result.Items.Should()
         .HaveCount(companiesSet.Count)
         .And.Contain(x => companiesSet.Any(c => c.Name == x.Name))
         .And.BeInDescendingOrder(x => x.Name);
```

B)
```csharp
result.Should()
	  .NotBeNull();
result!.Pager
	   .Count
	   .Should()
	   .Be(companiesSet.Count);
result.Items
	  .Should()
	  .HaveCount(companiesSet.Count).And
	  .Contain(x => companiesSet.Any(c => c.Name == x.Name)).And
	  .BeInDescendingOrder(x => x.Name);
```

https://csharpier.com/ for example would create the output at example A with it's default settings as tested on https://playground.csharpier.com/

I don't want to impose my opinion though, just wanted to share my thoughts on the formatting change. It's up to you if you prefer one or the other.

Reply:
Yeah, there's a number of ways in which it all can be formatted, but I think I'll leave this discussion for another issue so we can better analyze it and think about it and mark this conversation as resolved in this PR so we can merge it.
I still think it's important to be consistent and coherent when we define this, as someone might find it kind of capricious if we format things one way in a place and another way somewhere else.

Reply:
BTW, please feel free to open an issue for tracking the formatting stuff with your suggestions! 😉